src/server/routes/system/health.ts

```diff
@@ -61,7 +63,8 @@ export async function healthCheck(fastify: FastifyInstance) {
 }
 
 const getFeatures = (): EngineFeature[] => {
-  const features: EngineFeature[] = [];
+  // IP Allowlist is always available as a feature, but added as a feature for backwards compatibility.
```

Comment:
This comment isn't providing much value. Let's remove.

Reply:
As I understand the other feature flags are used to gate access, but `IP_ALLOWLIST` is being used for backwards compatibility, so wanted to add in explanation for why its in `features` in case it wasn't clear